### PR TITLE
Triplet vcpkg build instruction update

### LINKED ISSUE #5631: Mention host / target triplets in VCPKG tutorial
Is this something we should put into the install-PCL-via-VCPKG tutorial?

_Originally posted by @mvieth in https://github.com/PointCloudLibrary/pcl/pull/5614#pullrequestreview-1310208954_

### PR DESCRIPTION
**Pull request by rkscodes**
*fixes #5631*

### CODE REVIEW

File: doc/tutorials/content/pcl_vcpkg_windows.rst [hunk 1]
```diff
@@ -74,6 +74,16 @@ will install PCL with default options as well as default triplet type (ie. x86).
 
   ./vcpkg install pcl
   
+.. note::
+  This will build executables 2 times in release mode, as default host triplet is x64-windows
+  in various executables, but we use x86 by default. So to fix it you can provide default
```
Comment (larshg) [inline doc/tutorials/content/pcl_vcpkg_windows.rst:79]:
```suggestion
  on most modern PC systems, but vcpkg install x86 by default. So to fix it you can set
```

File: doc/tutorials/content/pcl_vcpkg_windows.rst [hunk 2]
```diff
@@ -74,6 +74,16 @@ will install PCL with default options as well as default triplet type (ie. x86).
...
+  host-triplet same as default triplet.
+
+    ./vckpgk install pcl --host-triplet -x86
```
[the remark this answers is not in the document]
Comment (larshg) [reply doc/tutorials/content/pcl_vcpkg_windows.rst:82]: Also don't you need to specify the full x86-windows ?

[the remark this answers is not in the document]
Comment (rkscodes) [reply doc/tutorials/content/pcl_vcpkg_windows.rst:82]: Yeah sure, it should be `x86-windows` without dash

File: doc/tutorials/content/pcl_vcpkg_windows.rst (hunk 1 above)
Comment (rkscodes) [reply doc/tutorials/content/pcl_vcpkg_windows.rst]: Updated

**Review by larshg — APPROVED** [review]

File: doc/tutorials/content/pcl_vcpkg_windows.rst [hunk 3]
```diff
@@ -74,6 +74,16 @@ will install PCL with default options as well as default triplet type (ie. x86).
 
   ./vcpkg install pcl
   
+.. note::
+  This will build executables 2 times in release mode, as default host triplet is x64-windows
+  on most modern PC systems, but vcpkg install x86 by default. So to fix it you can set
+  host-triplet same as default triplet.
+
+    ./vckpgk install pcl --host-triplet x86-windows
```
Comment (mvieth) [inline doc/tutorials/content/pcl_vcpkg_windows.rst:82]:
```suggestion
    ./vcpkg install pcl --host-triplet x86-windows 
```

File: doc/tutorials/content/pcl_vcpkg_windows.rst [hunk 4]
```diff
@@ -74,6 +74,16 @@ will install PCL with default options as well as default triplet type (ie. x86).
...
+.. note::
+  This will build executables 2 times in release mode, as default host triplet is x64-windows
+  on most modern PC systems, but vcpkg install x86 by default. So to fix it you can set
+  host-triplet same as default triplet.
+
+    ./vckpgk install pcl --host-triplet x86-windows 
+
+  Or, you can use same custom triplet for both --triplet and --host-triplet
+
+    ./vckpgk install pcl --triplet <same_custom_triplet_type> --host-triplet <same_custom_triplet_type>
```
Comment (mvieth) [inline doc/tutorials/content/pcl_vcpkg_windows.rst:86]:
```suggestion
    ./vcpkg install pcl --triplet <same_custom_triplet_type> --host-triplet <same_custom_triplet_type>
```

**Review by mvieth — APPROVED** [review]

**Pull request merged**